components/prism-bro.js

```diff
@@ -0,0 +1,102 @@
+Prism.languages.bro = {
+    'comment': [
+        {
+	        pattern: /(^|[^\\$])#.*/,
+		            	lookbehind: true
+		            }
+	],
+
+    'string': /(["'])(\\(?:\r\n|[\s\S])|(?!\1)[^\\\r\n])*\1/,
+
+    'boolean': /\b(T|F)\b/,
+
+    'keyword': [
+        /\b(break|next|continue)\b/,
+        /\b(alarm|using|of|add|delete)\b/,
+		/\b(export)\b/,
+        /\b(print|return|schedule)\b/,
+		/\b(when|timeout)\b/,
+		/\b(addr|any|bool|count)\b/,
+		/\b(double|enum)\b/,
+		/\b(file|int|interval)\b/,
+		/\b(pattern|opaque)\b/,
+		/\b(port|record|set)\b/,
+		/\b(string|subnet|table)\b/,
+		/\b(time|vector)\b/,
+		/\b(for|if|else)\b/,      
+	],
+	
+	'operator': /--?|\+\+?|!=?=?|<=?|>=?|==?=?|&&|\|\|?|\?|\*|\/|~|\^|%/,
+	
+	'function': [
+	{
+		pattern: /function [a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)?/i,
+		inside: {
+			keyword: /function/
+		}
+	},
+	
+	{
+		pattern: /hook [a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)?/i,
+		inside: {
+			keyword: /hook/
+		}
+	},
+	
+	{
+		pattern: /event [a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)?/i,
+		inside: {
+			keyword: /event/
+		}
+	}
+	],
+	
+	'variable':	
+	[{
+		pattern: /global [a-zA-Z0-9_]+/i,
+		inside: {
+			keyword: /global/
+		}
+	},
+	{
+		pattern: /local [a-zA-Z0-9_]+/i,
+		inside: {
+			keyword: /local/
+		}
+	}],
+	
+	
+	'number': [       
+	     /\b-?(?:0x[\da-f]+|\d*\.?\d+(?:e[+-]?\d+)?)\b/i,
+	     
+    ],
+    
+    'italic':  /\b(TODO|FIXME|XXX)\b/,
+	'punctuation': /[{}[\];(),.:]/,
+	
+	'builtin': 
+	[
+	     /@load\s+/,
+	     /@load-sigs\s+/,
```

Comment:
`@load-sigs` does not work, because the `-` sign is matched first as an operator and prevents the match as `@load-sigs`. You probably have to reorder `operator` and `builtin` in some manner for this to work.